src/puppetlabs/puppetdb/scf/storage_utils.clj

```diff
@@ -86,6 +86,25 @@
                    "INNER JOIN pg_namespace pn ON (pp.pronamespace = pn.oid) "
                    "INNER JOIN pg_language pl ON (pp.prolang = pl.oid) "
                    "WHERE pl.lanname NOT IN ('c') "
+                   "AND pp.proisagg = 'f'"
```

Comment:
Looks like _proisagg_ was replaced by _prokind_ in [PG 11+](https://www.postgresql.org/docs/11/release-11.htm) which looks like why some of the travis cells are failing. Maybe we can use the **db-metadata** func in storage-utils to sniff out the version of PG we're using and have these funcs switch the column name when needed.